qns/Dockerfile

```diff
@@ -10,10 +10,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
  && apt-get autoremove -y && apt-get clean -y \
  && rm -rf /var/lib/apt/lists/*
 
+ARG RUST_VERSION
+
 ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
-    PATH=/usr/local/cargo/bin:$PATH \
-    RUST_VERSION=1.45.2
+    PATH=/usr/local/cargo/bin:$PATH
 
 RUN set -eux; \
     curl -sSLf "https://static.rust-lang.org/rustup/archive/1.22.1/x86_64-unknown-linux-gnu/rustup-init" -o rustup-init; \
```

Comment:
It might be a good idea to update the version of rustup-init we use.

Reply:
:+1: 5d10edeb4fb2414debe4df774f3c3fd3cba86a64